src/visualizations/config/adapters/dhis_highcharts/series/gauge.js

```diff
@@ -17,9 +16,7 @@ export default function(series, dashboard, layout, legendSet) {
                 borderWidth: 0,
                 verticalAlign: 'bottom',
                 style: {
-                    fontSize: dashboard
```

Comment:
As this is the only place `dashboard` is used the param should be removed from the function and this should stop passing it https://github.com/dhis2/analytics/blob/a1e3fe5cdedd0e4f4d73528f7c5b461b033fb6a9/src/visualizations/config/adapters/dhis_highcharts/series/index.js#L135